app/scripts/views/dialog_new_post.coffee

```diff
@@ -19,7 +19,29 @@ module.exports = class DialogNewPostView extends DialogView
 
     @$('#post-form-tabs [href="#post-form-preview"]').on 'show', (e) ->
       formattedText = formatters.format $('#post-form-text').val(), 'markdown'
-      $('#post-form-preview').html(formattedText)
+
+      htmlI = (c) ->
+        ' <i class="' + c + '"></i>'
+
+      htmlA = (h,t,c) ->
+        ' <a class="' + c + '" href="' + h + '">' + t + '</a>'
+
+      splitTrimFilter = (t,s) ->
+        ( e.trim() for e in t.split s ).filter (e) ->
+          return e.length > 0
+
+      tags = splitTrimFilter $('#post-form-tags').val(), ','
+      clubs = splitTrimFilter $('#post-form-clubs').val(), ','
+
+      tagsHtml = ''
+      if tags.length > 0
+        tagsHtml = htmlI('icon-tags') + ( htmlA '/t/' + e, e, 'post-tag' for e in tags ).join(', ')
+
+      clubsHtml = ''
+      if clubs.length > 0
+        clubsHtml = htmlI('icon-group') + ( htmlA '/c/' + e, e, 'post-club' for e in clubs ).join(', ')
+
+      $('#post-form-preview').html('<div class="post-body">' + formattedText + '</div><div class="post-footer">' + tagsHtml + clubsHtml + '</div>')
```

Comment:
I think it would be cleaner to use jquery helpers to construct html nodes. Like this: `$("<div/>").addClass("post-body").html(formattedText)`